PCampReview.py

```diff
@@ -158,15 +155,80 @@ def infoPopup(self,message):
     messageBox = qt.QMessageBox()
     messageBox.information(None, 'Slicer mpMRI review', message)
 
+  def setupIcons(self):
+
+    def createQIconFromPath(path):
+      return qt.QIcon(qt.QPixmap(path))
+
+    iconPath = self.resourcesPath + 'Icons/'
+    self.dataSourceSelectionIcon = createQIconFromPath(iconPath +'icon-dataselection_fit.png')
+    self.studySelectionIcon = createQIconFromPath(iconPath +'icon-studyselection_fit.png')
+    self.seriesSelectionIcon = createQIconFromPath(iconPath + 'icon-seriesselection_fit.png')
+    self.segmentationIcon = createQIconFromPath(iconPath + 'icon-segmentation_fit.png')
+    self.completionIcon = createQIconFromPath(iconPath + 'icon-completion_fit.png')
+
+
+  def setupTabBarNavigation(self):
+    self.tabWidget = qt.QTabWidget()
+    self.layout.addWidget(self.tabWidget)
+    self.tabBar = self.tabWidget.childAt(1, 1)
+
+    self.dataSourceSelectionGroupBox = qt.QGroupBox()
+    self.studySelectionGroupBox = qt.QGroupBox()
+    self.seriesSelectionGroupBox = qt.QGroupBox()
+    self.segmentationGroupBox = qt.QGroupBox()
+    self.completionGroupBox = qt.QGroupBox()
+
+    self.dataSourceSelectionGroupBoxLayout = qt.QFormLayout()
+    self.studySelectionGroupBoxLayout = qt.QFormLayout()
+    self.seriesSelectionGroupBoxLayout = qt.QFormLayout()
+    self.segmentationGroupBoxLayout = qt.QFormLayout()
+    self.completionGroupBoxLayout = qt.QFormLayout()
+
+    self.dataSourceSelectionGroupBox.setLayout(self.dataSourceSelectionGroupBoxLayout)
+    self.studySelectionGroupBox.setLayout(self.studySelectionGroupBoxLayout)
+    self.seriesSelectionGroupBox.setLayout(self.seriesSelectionGroupBoxLayout)
+    self.segmentationGroupBox.setLayout(self.segmentationGroupBoxLayout)
+    self.completionGroupBox.setLayout(self.completionGroupBoxLayout)
+
+    size = qt.QSize()
+    size.setHeight(50)
+    size.setWidth(110)
+    self.tabWidget.setIconSize(size)
+
+    self.tabWidget.addTab(self.dataSourceSelectionGroupBox,self.dataSourceSelectionIcon, '')
+    self.tabWidget.addTab(self.studySelectionGroupBox, self.studySelectionIcon, '')
+    self.tabWidget.addTab(self.seriesSelectionGroupBox, self.seriesSelectionIcon, '')
+    self.tabWidget.addTab(self.segmentationGroupBox, self.segmentationIcon, '')
+    self.tabWidget.addTab(self.completionGroupBox, self.completionIcon, '')
+    self.tabWidget.connect('currentChanged(int)',self.onTabWidgetClicked)
+
+    self.setTabsEnabled([1,2,3,4], False)
+
+  def onTabWidgetClicked(self, currentIndex):
+    if currentIndex == 0:
+      self.onStep1Selected()
+    if currentIndex == 1:
+      self.onStep2Selected()
+    if currentIndex == 2:
+      self.onStep3Selected()
+    if currentIndex == 3:
+      self.onStep4Selected()
+    if currentIndex == 4:
+      self.onStep5Selected()
+
   def setup(self):
     # Instantiate and connect widgets ...
```

Comment:
You need to call setup on the parent class, see as an example https://github.com/fedorov/MyFirst3DSlicerExtensionTutorial/blob/master/Part1snapshot/LineIntensityProfile/LineIntensityProfile.py#L39